cs/v1/README.md

```diff
@@ -1,8 +1,98 @@
 # Commercial Schedule (CS) API
 
+The DCSA Commercial Schedules API is specified on [**SwaggerHub**](https://app.swaggerhub.com/apis/dcsaorg/DCSA_CS).
 
+### <a name="v100B1"></a>[Release v1.0.0 (24 September 2024)](https://app.swaggerhub.com/apis-docs/dcsaorg/DCSA_CS/1.0.0)
```

Comment:
```suggestion
### <a name="v100"></a>[Release v1.0.0 (24 September 2024)](https://app.swaggerhub.com/apis-docs/dcsaorg/DCSA_CS/1.0.0)
```

Reply:
I assume it is the final ;) 100B1 refers to 100 Beta 1 ;-)

Reply:
As a FYI - this also needs to be updated in the YAML file so it links forth and back

Reply:
Thanks Henrik for pointing that out. Updated the API as well